routes/products.js

```diff
@@ -0,0 +1,48 @@
+let express = require('express');
+let router = express.Router();
+let prodListing = require('../public/data/products/index.get.json');
+let prodCategories = require('../public/data/categories/index.get.json');
+let totalCounter = require('./data/constant');
+let commonContent = require('./data/strings');
+
+/* GET product listing. */
+router.get('/', function(req, res, next){    
+    ActiveCategories = prodCategories.filter(category => category.enabled);
```

Comment:
what is ActiveCategories? A variable?
should have proper scope as const or let, use camelCasing